### DISCUSSION TITLE
How to make full width header and full width all sites

### DISCUSSION THREAD

**Discussion by agriharyus** [bootscore/bootscore#748] [Q&A]
Anyone can help me please?
![Screenshot 2024-04-17 064103](https://github.com/bootscore/bootscore/assets/35243412/c7c1bde2-94bb-4814-a8d4-f8c8e07d500e)

**Comment by agriharyus** [comment]
I want to get rid of the spaces I marked throughout the site.

**Reply by crftwrk** [reply]
Use a function for the containers https://bootscore.me/documentation/containers/
